app/src/main/java/com/onegini/mobile/exampleapp/network/fcm/FCMRegistrationService.java

```diff
@@ -43,8 +42,8 @@ public FCMRegistrationService(final Context context) {
 
   public void enrollForPush(final OneginiMobileAuthWithPushEnrollmentHandler enrollmentHandler) {
     FirebaseApp.initializeApp(context);
-    final String fcmRefreshToken = FirebaseInstanceId.getInstance().getToken();
-    if (fcmRefreshToken == null) {
+    final String fcmRefreshToken = getStoredRegistrationId();
```

Comment:
Not sure if this change is valid. We wanted to double check that the app has a valid FCM token, which can be only done by calling FirebaseInstanceId. The local storage has "the last known" Token but it's not aware if the token wasn't revoked or if the Token update operation has failed etc.